uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/ForcedPasswordChangeIT.java

```diff
@@ -125,7 +125,7 @@ public void testHandleForceChangingPasswordWithNewPasswordSameAsOld() throws Exc
         webDriver.findElement(By.xpath("//input[@value='Create new password']")).click();
         assertEquals(baseUrl+"/force_password_change", webDriver.getCurrentUrl());
         assertThat(webDriver.findElement(By.cssSelector(".error-message")).getText(),
-            containsString("Your new password cannot be the same as the old password."));
+            containsString("Your new password cannot be the same as one in your recent password history."));
```

Comment:
Not really sure how to run the ITs.  Can someone send a link with instructions?  Thanks.